### PR TITLE
packages: dependencies reorganized and re-sort

### PR DESCRIPTION
**Pull request by Andarwinux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,90 +158,82 @@ to update flags which will pass on gcc, g++ and etc.
...
-    - xvidcore (1.3.7)
-    - lzo (2.10)
-    - libopenmpt (0.7.3)
-    - libiconv (1.17)
     - gmp (6.3.0)
+    - libiconv (1.17)
+    - libopenmpt (0.7.3)
+    - lzo (2.10)
     - vapoursynth (R65/R63)
-    - ~~mbedtls (3.5.0)~~
```
Comment (Kosm8) [inline README.md:243]: mbedtls is used by curl.

Comment (Andarwinux) [reply README.md:243]: curl is unused.

Comment (Kosm8) [reply README.md:243]: ok.

File: README.md [hunk 2]
```diff
@@ -158,90 +158,82 @@ to update flags which will pass on gcc, g++ and etc.
...
 - Zip
-    - xvidcore (1.3.7)
-    - lzo (2.10)
-    - libopenmpt (0.7.3)
-    - libiconv (1.17)
     - gmp (6.3.0)
+    - libiconv (1.17)
+    - libopenmpt (0.7.3)
+    - lzo (2.10)
     - vapoursynth (R65/R63)
```
Comment (NotTsunami) [inline README.md]:
```suggestion
    - vapoursynth (R65 for 64-bit compilation/R63 for 32-bit compilation)
```
Might be better to be more verbose?

Comment (Andarwinux) [reply README.md]: thanks

**Comment by Kosm8** [comment]
Can you consider giving each package its own line?

**Comment by shinchiro** [comment]
Since you want to organize the packages' list, put it in table since its getting longer
|    |     |     |    |    |
|---|---|---|---|---|
| package1 | package2 | package3 | package4 | package5 | 
| package6 | package7 | package8 | package9 | package10 | 

Code:
```
|  |  |  |  |  |
|---|---|---|---|---|
| package1 | package2 | package3 | package4 | package5 | 
| package6 | package7 | package8 | package9 | package10 | 
```

**Comment by Kosm8** [comment]
The dependency graph is a good idea. It doesn't display well on my screen, but the raw text is useful.

**Comment by zhongfly** [comment]
> what do you think about this?

Same as your opinion, using table makes adding and deleting more difficult.And I don't think table looks good.If it's too long, we can put it in the click-to-expand section.

> Will this change cause problems with the README for mpv-winbuild

No effect on my repo.

**Comment by rach-md** [comment]
xxhash --> aom to dav1d.
lcms2, zlib is not explicitly enabled in mpv.cmake but still autodetect,  I guess.
I can't find anywhere ffmpeg and mpv needed libpng as their dependency
vulkan package also is not explicitly added in ffmpeg.cmake.
what would you do with packages that not related to mpv build?
personally not really into graph addition btw

**Comment by shinchiro** [comment]
Frankly, I never feel to merge this since I dont want to open a can of worms. After all, I will need to fix all the bugs happen in future which I'm lazy. The way it is now is good enough. The only things I dont like is unnecessary long list of packages. Maybe I will delete them for good?

**Pull request closed without merging**